preview/action.yaml

```diff
@@ -10,40 +10,41 @@ inputs:
   project-slug:
     description: "Project's slug on Read the Docs"
     required: true
-  project-language:
+  project-languages:
```

Comment:
It would be good to keep the `project-language` attribute to avoid a breaking change here. We can probably add `project-languages` (with "s") as a new field and mark the old one as deprecated.